src/auth/backchannel.ts

```diff
@@ -98,7 +98,12 @@ export type AuthorizeOptions = {
    * Optional parameter for subject issuer context.
    */
   subjectIssuerContext?: string;
-};
+  /**
+   * Optional authorization details to use Rich Authorization Requests (RAR).
+   * @see https://auth0.com/docs/get-started/apis/configure-rich-authorization-requests
+   */
+  authorization_details?: string;
```

Comment:
@frederikprijck What’s the reason for exposing this argument as a string?

Wouldn’t it be better to define it as a JSON object and save the developer from the extra step of having to call `JSON.stringify(authorization_details)`? Or am I missing something?

Asking because we are about to add this parameter in the auth0-ai-* packages.